_episodes/57.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: episode
+title: "57: Seeing clearly with OpenTelemetry"
+date: 2024-03-14
+tags: trino opentelemetry openmetrics observability metrics
+youtube_id: "ofNw2FErJpM"
+wistia_id: "00xhwailn3"
+sections:
+- title: "Introduction"
+  desc: "Episode 57: Seeing clearly with OpenTelemetry"
+  time: 0:00
+- title: "Releases"
+  desc: "Trino 440, Trino 441 and other updates"
+  time: 1:15
+- title: "Guest introductions"
+  desc: "David Phillips and Matt Stephenson"
+  time: 4:02
+- title: File system lead and related API and features
+  time: 5:48 File system lead and related API and features
+- title: Java 22
+  desc: Upcoming usage with Trino
+  time: 9:30 
+- title: Japanese edition of Trino The Definitive Guide
+  desc: Props to Yuya Ebihara 
+  time: 11:06 
+- title: Trino Contributor Call 
+  time: 12:03 
+- title: OpenTelemetry and OpenMetrics introduction and concepts 
+  time: 12:27 
+- title: Motivation to add to Trino
+  time: 15:32
+- title: Why is tracing useful in Trino?
+  time: 17:23 
+- title: Use in development
+  time: 18:57
+- title: Implementation details
+  time: 19:59
+- title: Trace propagation from clients and elsewhere
+  time: 23:55
+- title: Practical experience and demo
+  time: 27:44
+- title: OpenMetrics
+  time: 37:54
+- title: Prometheus sidecar, Airlift, Trino Gateway and more OpenMetrics
+  time: 40:04 
+- title: OpenMetrics vs OpenTelemetry metrics
+  time: 45:23 
+- title: JSON-format logging with TCP socket streaming
+  time: 48:53
+- title: Learning resources
+  desc: 
+  time: 50:15
+- title: Trino Fest and Trino Contributor Congregation
+  time: 52:50
+- title: Upcoming Trino Community Broadcast episodes
+  time: 54:55
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Director of Trino
+  Community Leadership at [Starburst](https://starburst.io),
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [David Phillips](https://www.linkedin.com/in/electrum/), co-creator of Trino
+  and CTO at [Starburst](https://starburst.io)
+* [Matt Stephenson](https://www.linkedin.com/in/jmstephenson/), Senior Principal
+  Software Engineer at [Starburst](https://starburst.io)
+
+## Trino releases
+
+[Trino 440](https://trino.io/docs/current/release/release-440.html)
+
+* New Snowflake connector
+* Support for sub-queries inside `UNNEST` clauses
+* Support for row filtering and column masking with Open Policy Agent
+* Improved latency when filesystem caching is enabled in Delta and Iceberg connectors
+
+[Trino 441](https://trino.io/docs/current/release/release-441.html)
+
+* Remove the default `legacy` mode for `hive.security`
+
+And there is a regression for Iceberg, so wait for 442 potentially.
```

Comment:
Do we want to call out that 442 is out already?

Reply:
yeah .. let me do that..